CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
 # Changelog
+### 2.23.3
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.8.15](https://fluentbit.io/announcements/v1.8.15/)
+* Amazon CloudWatch Logs for Fluent Bit 1.7.0
+* Amazon Kinesis Streams for Fluent Bit 1.9.0
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Same as `2.23.2`, this release includes the following fix for AWS customers that we are working on getting accepted upstream:
+* Bug - Resolve IMDSv1 fallback error introduced in 2.21.0 [aws-for-fluent-bit:259](https://github.com/aws/aws-for-fluent-bit/issues/259)
+
+Note:
+* This release includes several core fixes added in Fluent Bit 1.8.15 including enhanced networking thread safety, and keep_alive connection error handling.
```

Comment:
I don't think we need this honestly. I know there isn't anything new except the bump to 1.8.15, but that's already called out at the top with a link to the release notes. We don't have any specific data indicating that these fixes affect AWS customers, we're just putting them out in case they do. THus, I think this call out is in some sense misleading since it implies we have reason to call out these fixes.

Reply:
Removed.